CHANGELOG.md

```diff
@@ -24,11 +25,23 @@ and includes an additional section for migration notes.
 - *ORCA-597*
   - Server access logging is now enabled for graphql application load balancer.
 
+### Changed
+- *ORCA-573* Updated ORCA DB user password to now have a stronger password requirement. See migration notes for details.
+
+
 ### Migration Notes
 - Update the bucket policy for your `system-bucket` to allow load balancer to post server access logs to the bucket. See the instructions [here](https://nasa.github.io/cumulus-orca/docs/developer/deployment-guide/deployment-s3-bucket#bucket-policy-for-load-balancer-server-access-loging).
 - InternalReconcileReport [Phantom](https://nasa.github.io/cumulus-orca/docs/developer/api/orca-api#internal-reconcile-report-phantom-api) and [Mismatch](https://nasa.github.io/cumulus-orca/docs/developer/api/orca-api#internal-reconcile-report-mismatch-api) reports are now available via GraphQL.
   - API Gateway access is now deprecated, and will be removed in a future update.
   - Use the `orca_graphql_load_balancer_dns_name` variable to send your queries to GraphQL as json strings in a POST request.
+- Users will need to update their orca-user password. The password must have the following requirements otherwise the `db_deploy` lambda will fail during deployment.
+  - one upper case letter
+  - one lower case letter
+  - one digit
+  - one special character
+  - minimum length of 12
+
+  Update `db_user_password` variable in your `cumulus-tf/terraform.tfvars` file to match the new password requirement and then run terraform.
```

Comment:
I don't think this will be enough, unfortunately. db_deploy doesn't recreate the user account on every TF apply.
Try changing your password and then hitting one of the API Gateway reporting endpoints. My guess is that you'll see a permissions/login error.

Reply:
hmm looks like it throws autentication error
`FATAL:  password authentication failed for user \"rhrh_orcauser`

Reply:
Yup. You'll need something that can reset the password on that user. You can either create manual steps or, my preferred solution, have db_deploy reset that users password. Probably as the first step in this block: https://github.com/nasa/cumulus-orca/blob/ca65aaf37f0738540f4f343407231da42e84fbba/tasks/db_deploy/db_deploy.py#L104
Without trying it myself, I believe the syntax you need is "ALTER ROLE "{user_name}" WITH ENCRYPTED PASSWORD :user_password;" https://www.postgresql.org/docs/current/sql-alterrole.html

Reply:
ok I am currently looking into updating db_deploy to reset the password using that `alter role` command above. 
Will let you know the outcome

Reply:
Looked a little closer, and if this plan does work, it would actually go at the top of the the `if app_schema_exists` block. With my initial suggestion you'd have this new code running before the user it modifies is created in `create_fresh_orca_install`.

Reply:
tried the above method. Check this 
https://github.com/nasa/cumulus-orca/pull/387